time_tracker_api/users/users_namespace.py

```diff
@@ -57,7 +66,11 @@ class Users(Resource):
     @ns.marshal_list_with(user_response_fields)
     def get(self):
         """List all users"""
-        return AzureConnection().users()
+        ftm = FeatureToggleManager('bk-user-role-field')
+        if ftm.is_toggle_enabled_for_user():
+            return AzureConnection().users_v2()
+        else:
+            return AzureConnection().users()
```

Comment:
Maybe we could avoid that `else` by the short circuit that the return inside the `if` clause is doing. :)

Reply:
It is ok, can be reduced.